Package.swift

```diff
@@ -28,7 +28,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/apple/swift-atomics.git", from: "1.2.0"),
         .package(url: "https://github.com/apple/swift-algorithms.git", from: "1.2.0"),
-        .package(url: "https://github.com/StanfordBDHG/XCTRuntimeAssertions.git", from: "1.1.3")
+        .package(url: "https://github.com/StanfordBDHG/XCTRuntimeAssertions.git", revision: "a41e7f4991102d01e76897f9d016f3efe0557bac")
```

Comment:
might wanna tag a proper release here instead of hard-coding the current main branch revision

Reply:
Yes, please tag a semantic version appropriate release in the repo 👍